bin/set-major-version-tag.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
+const { exec } = require('child_process');
+
+const semverParse = require('semver/functions/parse');
+
+const pkg = require('../package.json');
+
+/**
+ * Sets major git tag as part of release process
+ *
+ * @example v7
+ */
+async function setMajorVersionTag() {
+  const parsedVersion = semverParse(pkg.version);
+
+  if (parsedVersion.prerelease.length) {
+    // eslint-disable-next-line no-console
+    console.warn('Pre-release version, not setting major version tag');
+    return;
+  }
```

Comment:
We could use our `getPkgVersion` and `getMajorPkgVersion` functions here, but I wanted to keep this fairly vanilla so we can refer to it for a hypothetical future blog post. Plus we need some semver logic here that is pretty specific.